html/index.html

```diff
@@ -89,6 +94,9 @@
                                 appType: data.name,
                             });
                         }
+                    },
+                    translate(phrase) {
+                        return viewmodel.translations[phrase] || phrase;
```

Comment:
```suggestion
                        return viewmodel.translations[phrase] || `Could not find phrase: ${phrase}`;
```
Better value for non found translations